docs/settings.rst

```diff
@@ -1007,7 +1007,7 @@ this to an empty string disables secpoll.
 -  String
 -  Default: The hostname of the server
 
-This is the server ID that will be returned on an EDNS NSID query.
+This is the server ID that will be returned on an EDNS NSID query. It defaults to the server's hostname.
```

Comment:
this is already mentioned two lines above?

Reply:
If only I could read ...